docs/docs/signing.md

```diff
@@ -79,84 +99,63 @@ const req = {
 const sig = await client.sign(req)
 ```
 
-### 💾 Calldata Decoding
+## 💾 Calldata Decoding
 
-:::note
-All available calldata decoding options will be documented in this section. More may be added as time goes on.
-:::
+> NOTE: All available calldata decoding options will be documented in this section. More may be added as time goes on.
 
 Certain transaction decoder types may support calldata decoding for request data. You can use this feature by including "calldata decoder data" (explained shortly) in a general signing request using the `decoder` request param:
 
 ```ts
-req.decoder = <calldata decoder data>
+req.data = {
+  payload: <Raw message to be signed, e.g. serialized transaction>,
+  decoder: <Optional serialized information about decoding the payload>
+}
 await client.sign(req);
 ```
 
 If you include a valid calldata decoder, the appearance of the transaction's data on the user's Lattice should transform from a raw hex string to a markdown-style version which displays the function name, parameter names, and values.
 
-#### Storing Calldata Decoders
+### 1️⃣ EVM Calldata Decoding
 
-Although not necessary, in certain situations it may be advantageous to pre-save decoders to the Lattice. One advantage is that if the decoder is saved, you do not need to include it in the transaction request, which frees up some space. Additionally, pre-saving data may unlock certain security features depending on the decoder type.
+EVM transactions serialize calldata according to the [Ethereum ABI specification](https://docs.soliditylang.org/en/latest/abi-spec.html). The first four bytes of a transaction's `data` represent the "function selector", which is (sort of) a unique identifier for a given function. 
 
-You can use the following API:
+> NOTE: We do not support 100% of all edge cases in the ABI specification, but we do support the vast majority of types.  Please open a pull request or an issue if your request fails to decode on a Lattice.
```

Comment:
We could use the `:::note` syntax here

Reply:
[074aaa8](https://github.com/GridPlus/gridplus-sdk/pull/467/commits/074aaa820cb7ae45a3de58c7cbafbbc6c1fb8287)